pkg/projects/Microsoft.NETCore/Microsoft.NETCore.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <Version>5.0.3</Version>
+    <Version>5.0.2</Version>
```

Comment:
I think it is fine to keep it stable here but I do worry that we might end up building and shipping it from both these branches. So please make sure you test this and ensure that the package doesn't end up in the output folder otherwise it will get picked up and published.